pkg/workload/querybench/query_bench.go

```diff
@@ -168,29 +154,52 @@ func (g *queryBench) Ops(
 	return ql, nil
 }
 
-// GetQueries returns the lines of a file as a string slice. Ignores lines
-// beginning with '#' or '--'.
-func GetQueries(path string) ([]string, error) {
+// GetQueries returns the queries in a file as a slice of named statements. If
+// no separator is given, splits by newlines.
+func GetQueries(path, separator string) ([]namedStmt, error) {
 	file, err := os.Open(path)
 	if err != nil {
 		return nil, err
 	}
 	defer file.Close()
 
+	// Remove comments at start of line. Don't attempt to parse the query, just
+	// naïvely interpret # and -- as comments.
+	reComments := regexp.MustCompile(`(?m)^\s*(#|--).*`)
+	reNamedQuery := regexp.MustCompile(`(?s)^\s*(\S+):\s*(.*)$`)
```

Comment:
nit: Would be nice to have a comment what named query is or an example from hight level help e.g. // name: SELECT foo FROM bar

Reply:
Sure, done.